README.md

```diff
@@ -3,6 +3,9 @@
 
 [![Build Status](https://travis-ci.org/geddy/geddy.png?branch=master)](https://travis-ci.org/geddy/geddy) [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/geddy/geddy)
 
+## DEPRECATION NOTICE
+**Geddy is no longer actively maintained. It is not recommended for new project.**
```

Comment:
It sounds picky of me @phanect, but it is better English to say:
"Geddy is no longer actively maintained, and therefore it is not recommended to be used for any new projects."